src/components/game-page/rockscissorspaper-page/Timer.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import RockScissorsPaper from './RockScissorsPaper';
+
+const Timer = () => {
+  const [timer, setTimer] = useState(3);
+
+  const CountDown = setTimeout(() => {
+    setTimer(timer - 1);
+  }, 1000);
+
+  useEffect(() => {
+    timer === 0 ? clearTimeout(CountDown) : CountDown;
```

Comment:
```suggestion
    timer === 0 ? clearTimeout(CountDown) : CountDown();
```
뒷 CountDown은 호출이 필요해보이는데 상관없는걸까요?~!?

Reply:
늦게 봐서 죄송합니다 😱 🙀 💻 
지금은 수정되었습니다! 감사합니다 😄 😸 😃 😺